### DISCUSSION TITLE
When can we expect a release containing the vulnerability fix for golang.org/x/crypto?

### DISCUSSION THREAD

**Discussion by zz671** [nginx/nginx-prometheus-exporter#933] [Q&A] [answered]
golang.org/x/crypto has been flagged as a critical vulnerability for versions < 0.31.0. The main branch contains this updated dependency however the latest release 1.4.0 of the exporter is using v0.28.0.

**Comment by lucacome** [comment, marked as answer]
@zz671 https://github.com/nginx/nginx-prometheus-exporter/releases/tag/v1.4.1 was just released
